### PR TITLE
Added Insertion linked list java

### LINKED ISSUE #340: Insertion in LinkedList
# Description
, Linked List is a linear data structure. Unlike arrays, linked list elements are not stored at a contiguous location; the elements are linked using pointers.

# Additional context
Methods to insert a new node in linked list are to be implemented. A node can be added in three ways 
1) At the front of the linked list 
2) After a given node. 
3) At the end of the linked list.

# Important Checklist 
-   [. ] I have read the contribution guidelines before making this issue.
-   [. ] I have verified this issue program is not already implemented
-   [ .] I have not pushed this code anywhere else (even my repo)

# Languages and Assignees:
-   [x] Python: @chAlekhya 
-   [x] Java: @jindalsachin
-   [x] C: @muffpy 
-   [x] C++: @valar-yudha

### PR DESCRIPTION
**Pull request by sj538**
# Description

Insertion in  the LinkedList  for Java is  included in this PR

closes #340 

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

# Checklist:

-   [x] I have read the contribution guidelines before making this PR.
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented in my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@jindalsachin Fill PR template properly with appropriate issue number

**Comment by sj538** [comment]
@kaustubhgupta I have filled the PR template Please check

File: InsertionLinkedList.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/* A node can be added in three ways 
+1) At the front of the linked list 
+2) After a given node. 
+3) At the end of the linked list.
+
+I have implemented all 3 insertion in simple program with the hellp of 3 different classes respecively.
```
Comment (kaustubhgupta) [inline InsertionLinkedList.java]: - Delete this line
- Update java folder readme

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
https://www.geeksforgeeks.org/linked-list-set-2-inserting-a-node/
Not acceptable

**Pull request closed without merging**